yogstation/code/modules/jobs/security/aiming.dm

```diff
@@ -0,0 +1,206 @@
+/obj/item/gun/Initialize()
+	. = ..()
+	AddComponent(/datum/component/aiming)
+
+/obj/item/reagent_containers/food/snacks/grown/banana/Initialize() //Yes, you can hold someone at gunpoint with a banana.
+	. = ..()
+	AddComponent(/datum/component/aiming)
+
+/mob/living/proc/aim_react()
+	set waitfor = FALSE
+	var/list/options = list()
+	for(var/option in list("surrender", "ignore"))
+		options[option] = image(icon = 'yogstation/icons/effects/aiming.dmi', icon_state = option)
+	var/choice = show_radial_menu(src, src, options)
+	if(choice == "surrender")
+		emote("surrender")
+
+/datum/action/item_action/aim
+	name = "Manually aim"
+	desc = "Aim at a target manually. You can also aim with 'point' or SHIFT + middle mouse while holding a gun."
+	button_icon_state = "aim"
+	icon_icon = 'yogstation/icons/mob/actions/actions.dmi'
+
+/obj/item/ui_action_click(mob/user, action)
+	if(istype(action, /datum/action/item_action/aim))
+		var/datum/component/aiming/aiming = GetComponent(/datum/component/aiming)
+		if(!aiming)
+			return
+		var/list/mobs = list()
+		for(var/mob/living/M in view(user, 15))
+			if(M == user || !isliving(M) || M.invisibility > 0 || !M.alpha)
+				continue
+			mobs += M
+		var/mob/living/A = input(user, "Who do you want to point [src] at?", "[src]", null) as null|anything in mobs
+		if(isliving(A))
+			aiming.aim(user, A)
+		return
+	. = ..()
+
+//Happens before the actual projectile creation
+/obj/item/gun/before_firing(atom/target,mob/user, aimed)
+	if(aimed)
+		if(chambered?.BB && !istype(src, /obj/item/gun/ballistic/automatic/toy))
+			chambered.BB.stamina = initial(chambered.BB.stamina) += 55
```

Comment:
this happens

Reply:
ok this is a\ctually a massive bruh moment now it needs to not give stamina damage to everything
normal holdup gets away with boosted damage because it's both triggered automatically and requires time with this you can spam it

Reply:
Didn't realize it was giving the extra damage. It was under the code for the "make bullet a bit faster" section so I assumed it was something unique to non-lethal projectiles or something. I'll go remove it here in a bit, didn't intend to have any damage buffs in here anyways